npm-search/assets/js/app.js

```diff
@@ -168,7 +168,15 @@ var Plugin = React.createClass({
             </li>
         )
     }
-})
+});
+
+var timer=0;
```

Comment:
MAking it a member of PluginList class will make it a state variable meaning we will have to update it's state every time we render PluginList which is not really required.